admin/database/postgres/postgres.go

```diff
@@ -696,6 +696,11 @@ func (c *connection) DeleteProjectMemberUser(ctx context.Context, projectID, use
 	return checkDeleteRow("project member", res, err)
 }
 
+func (c *connection) DeleteAllProjectMemberUserForOrganization(ctx context.Context, orgID, userID string) error {
+	res, err := c.getDB(ctx).ExecContext(ctx, "DELETE FROM users_projects_roles upr WHERE upr.user_id = $1 AND upr.project_id IN (SELECT p.id FROM projects p WHERE p.org_id = $2)", userID, orgID)
+	return checkDeleteRow("project member", res, err)
```

Comment:
Use `parseErr`, this one only works when exactly 1 row will be deleted (in this case, 0 to many rows may be deleted)